### PR TITLE
[Clean] Use already integrated data

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

File: src/ride/ride.c [hunk 1]
```diff
@@ -6049,10 +6049,10 @@ static money32 ride_create(int type, int subType, int flags, int *outRideIndex,
 	ride->music = RCT2_ADDRESS(0x0097D4F4, uint8)[ride->type * 8];
 
 	ride->operation_option = (
-		RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + 4 + (ride->type * 8), uint8) +
-		RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + 4 + (ride->type * 8), uint8) +
-		RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + 4 + (ride->type * 8), uint8) +
-		RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + 5 + (ride->type * 8), uint8)
+		RideProperties[ride->type].min_value +
```
Comment (duncanspumpkin) [inline src/ride/ride.c]: Any idea what this is doing? Setting default value to something a bit lower than the average of min and max? Can you change it to a x3 to make it easier to read.

Comment (marijnvdwerf) [inline src/ride/ride.c]: Seems like a weighted average, leaning more towards the lower bound.

File: src/ride/ride.c [hunk 2]
```diff
@@ -3758,8 +3758,8 @@ static bool ride_is_valid_num_circuits(rct_ride *ride, int numCircuits)
 
 static bool ride_is_valid_operation_option(rct_ride *ride, uint8 value)
 {
-	uint8 minValue = RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + (ride->type * 8) + 4, uint8);
-	uint8 maxValue = RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + (ride->type * 8) + 5, uint8);
+	uint8 minValue = RideProperties[ride->type].min_value;
+	uint8 maxValue = RideProperties[ride->type].max_value;
 	if (ride->mode == RIDE_MODE_MAZE) {
```
Comment (Gymnasiast) [inline src/ride/ride.c]: Could you remove this if block as well? I missed it in #4205.

Comment (marijnvdwerf) [inline src/ride/ride.c]: Removed another one as well

**Pull request merged**